documents/operating.md

```diff
@@ -0,0 +1,216 @@
+Bad Idea Factory Limited Liability Corporation
+===================================================================
+
+Contents:
+
+1.  Official Operating Agreement
+2.  Partnership
+3.  Projects
+4.  Money
+5.  Governing Process
+6.  Policies
+
+---------------------------------------------------------------------
+SECTION 1: 📚📚📚 Official Operating Agreement 📚📚📚
+---------------------------------------------------------------------
+- [x] Company term
+- [x] Continuance of company
+- [x] Rickroll
+- [x] Game of Mario Party 3 before legal arbitration between members
+
+### Company Term
+BIFFUD will exist until either (1) a vote to terminate the organization
+is passed by a consensus of all Corporate Overlords or (2) the board fails to
+hold a single meeting with quorum for a period of (a consecutive year?)
```

Comment:
Should the organization cease to exist if a quorum cannot be reached for a sustained period of time?  Or should it shed members until it's down to the folks who show up to meetings?  Cause it seems weird that we could in theory grow to a size where quorums become impossible, and then the company simply ends after a year.

Or, put another way, should we first have a period of time where folks fall out of active full partnership, PRIOR to the org simply folding up and closing?